spec/integration/app_peeps_spec.rb

```diff
@@ -0,0 +1,104 @@
+require "spec_helper"
+require "rack/test"
+require_relative '../../app'
+require_relative '../reset_database_tables'
+
+describe Application do
+
+  before(:each) do
+    ResetDatabaseTables.new.reset
+  end
+
+  include Rack::Test::Methods
+
+  let(:app) { Application.new }
+
+  context 'GET /' do
+    it 'returns homepage' do
+      response = get('/')
+
+      expect(response.body).to include('Chitter</a></h1>')
+      expect(response.body).to include('peeps">See all Peeps!</a></h2>')
+      expect(response.body).to include('new">Post a Peep!</a></h2>')
+      expect(response.body).to include('>Sign up!</a></h2>')
+    end
+  end
+
+  context 'GET /peeps' do
+    it 'returns all peeps - latest peeps first' do
+      response = get('/peeps')
+
+      # extra spacing in next test to account for the spacing in peeps.erb
+      expect(response.body).to include("Latest peep is from:\n      @monica0")
```

Comment:
remember to test for response.status to check if the backend is working as expected.